### PR TITLE
Continue best practices for libpacemaker

### PR DESCRIPTION
**Pull request by kgaillot**
Bring remaining functions in pcmk_graph_producer.c up to current guidelines

### CODE REVIEW

**Comment by kgaillot** [comment]
@clumens , here's another low-priority review

File: lib/pacemaker/pcmk_graph_producer.c [hunk 1]
```diff
@@ -182,119 +182,131 @@ add_downed_nodes(xmlNode *xml, const pe_action_t *action,
...
         // CIB-only clean-up for shutdown locks
         action_xml = create_xml_node(NULL, XML_GRAPH_TAG_CRM_EVENT);
         crm_xml_add(action_xml, PCMK__XA_MODE, XML_TAG_CIB);
 
-/* 	} else if(pcmk__str_eq(action->task, RSC_PROBED, pcmk__str_casei)) { */
-/* 		action_xml = create_xml_node(NULL, XML_GRAPH_TAG_CRM_EVENT); */
+#if 0
+    } else if (pcmk__str_eq(action->task, RSC_PROBED, pcmk__str_none)) {
+        action_xml = create_xml_node(NULL, XML_GRAPH_TAG_CRM_EVENT);
+#endif
```
Comment (clumens) [inline lib/pacemaker/pcmk_graph_producer.c]: For blocks like this, I'd love a comment indicating why it is not currently enabled.

File: lib/pacemaker/pcmk_graph_producer.c [hunk 2]
```diff
@@ -182,119 +182,131 @@ add_downed_nodes(xmlNode *xml, const pe_action_t *action,
...
-        crm_xml_add(action_xml, XML_LRM_ATTR_TARGET_UUID, action->node->details->id);
-        if (router_node) {
-            crm_xml_add(action_xml, XML_LRM_ATTR_ROUTER_NODE, router_node->details->uname);
+        crm_xml_add(action_xml, XML_LRM_ATTR_TARGET,
+                    action->node->details->uname);
+        crm_xml_add(action_xml, XML_LRM_ATTR_TARGET_UUID,
+                    action->node->details->id);
+        if (router_node != NULL) {
+            crm_xml_add(action_xml, XML_LRM_ATTR_ROUTER_NODE,
+                        router_node->details->uname);
```
Comment (clumens) [inline lib/pacemaker/pcmk_graph_producer.c]: It wouldn't really save much (in fact it's implemented as crm_xml_add internally) but you could use pcmk__xe_set_propv to set the first two in one go.

File: lib/pacemaker/pcmk_graph_producer.c [hunk 3]
```diff
@@ -920,7 +918,7 @@ pcmk__graph_has_loop(pe_action_t *init_action, pe_action_t *action,
  * \return Newly added XML element for new graph synapse
  */
 static xmlNode *
-create_graph_synapse(pe_action_t *action, pcmk_working_set_t *data_set)
+create_graph_synapse(pe_action_t *action, pe_working_set_t *data_set)
```
Comment (clumens) [inline lib/pacemaker/pcmk_graph_producer.c:921]: You probably meant to merge this in with the previous patch.

File: lib/pacemaker/pcmk_graph_producer.c (hunk 1 above)
Comment (kgaillot) [reply lib/pacemaker/pcmk_graph_producer.c]: Agreed, added commit on top cleaning up the situation

File: lib/pacemaker/pcmk_graph_producer.c (hunk 2 above)
Comment (kgaillot) [reply lib/pacemaker/pcmk_graph_producer.c]: not a bad idea but probably not worth changing for just 2

File: lib/pacemaker/pcmk_graph_producer.c (hunk 3 above)
Comment (kgaillot) [reply lib/pacemaker/pcmk_graph_producer.c:881]: whoops :) updated commit

**Comment by clumens** [comment]
Looks good to me.

**Pull request merged**